### PR TITLE
Miscellaneous chores and clean-up

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline

File: tests/helpers.py [hunk 1]
```diff
@@ -62,11 +62,11 @@ def ip_to_device_id_map(device_group, devauth=devauth):
     @staticmethod
     def _check_log_for_message(device, message, since=None):
         if since:
-            cmd = f"journalctl -u mender-authd -l -S '{since}'"
+            cmd = f"journalctl --unit mender-authd --full --since '{since}'"
```
Comment (oleorhagen) [inline tests/helpers.py:65]: <3 Man of my heart

**Review by oleorhagen — APPROVED** [review]
Nice cleanups!

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline

**Pull request merged**